src/Umbraco.Web.UI/config/umbracoSettings.config

```diff
@@ -310,10 +310,7 @@
         Configure it here if you need anything specific. Needs to be a complete url with scheme and umbraco
         path, eg http://mysite.com/umbraco. NOT just "mysite.com" or "mysite.com/umbraco" or "http://mysite.com".
   -->
-  <web.routing
-    trySkipIisCustomErrors="false"
-    internalRedirectPreservesTemplate="false" disableAlternativeTemplates="false" disableFindContentByIdPath="false"
-    umbracoApplicationUrl="">
+  <web.routing trySkipIisCustomErrors="false" internalRedirectPreservesTemplate="false" disableAlternativeTemplates="false" disableFindContentByIdPath="false" umbracoApplicationUrl="" disableRedirectUrlTracking="true">

```

Comment:
Presume you did not mean to push this change @zpqrtbnk ? Can you clarify for me please :)